web/src/components/apps/Dashboard.jsx

```diff
@@ -781,9 +798,9 @@ class Dashboard extends Component {
             className="Modal DefaultSize"
           >
             <div className="Modal-body">
-              <p className="u-fontSize--large u-fontWeight--bold u-textColor--primary">
+              <ParagraphLarge weight={"bold"} theme={theme}>
                 Resource status
-              </p>
+              </ParagraphLarge>
```

Comment:
Here's an example of a styled component. If you look into our code base, `u-fontSize--large u-fontWeight--bold u-textColor--primary` is commonly used and can be replaced with ParagraphLarge

Reply:
this looks cool